examples/fastapi-crud/src/core/config.py

```
@@ -8,59 +8,59 @@
-    class Config:
-        env_file = ".env.database"
+   class Config:
+     env_file = ".env.database"
 
 
 class Settings:
 
-    @lru_cache
-    def get_api_settings() -> ApiSettings:
-        return ApiSettings()
+   @lru_cache
```

Ruff (0.12.2)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)